const handleError = (err)=>{
    console.log(err)
    throw err
}

export {
    handleError
}